CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## v3.7.0 / 2022-03-30
+
+Feature-level updates:
+- [#1989] Fix: resolve issues with metadata and the associated overriding behavior (thanks @maverick1872, @wbt, @DABH, @fearphage and issue reporters)
+
+Patch-level updates:
+- [#2075] Fix: add missing types for batching options for HTTP Transport (thanks @KylinDC)
+- Various dependencies updated, quality of life & maintainability changes, etc
```

Comment:
Fine for this one, but some additional detail (see past changelog examples) would probably be better in the future, like listing the PR #s for dependency bumps and noting the issue templates one separately.